shared/api/url.go

```diff
@@ -57,7 +57,7 @@ func (u *URL) Project(projectName string) *URL {
 
 // Target sets the "target" query parameter in the URL if the clusterMemberName is not empty or "default".
 func (u *URL) Target(clusterMemberName string) *URL {
-	if clusterMemberName != "" {
+	if clusterMemberName != "" && clusterMemberName != "none" {
```

Comment:
Please can you add a comment as to how this can come about? Where does "none" come from?

Reply:
So prior to the changes to the generator that were just pushed, when `UsedBy` fields were populated for Projects, we were stripping out `target=none` and `project=default`. I just moved it over assuming it was important:

```go
	for j := range value {
			if len(value[j]) > 12 && value[j][len(value[j])-12:] == "&target=none" {
				value[j] = value[j][0 : len(value[j])-12]
			}
			if len(value[j]) > 16 && value[j][len(value[j])-16:] == "?project=default" {
				value[j] = value[j][0 : len(value[j])-16]
			}
		}
```

You can see the diff here: https://github.com/lxc/lxd/commit/13def65775a19f38fd268ad720605a97e3c3d282#diff-5b408e12c31e2c3614dce882c6e52920de957b2565c76bbd86b06a940eea1d0a 
under `lxd/db/projects.mapper.go` line 161.

I think it's for storage pools/volumes, because we're filtering out `nodesName[*r.nodeID] == "none"` in `lxd/db/storage_pools.go`:
https://github.com/lxc/lxd/blob/master/lxd/db/storage_pools.go#L138-L153

Reply:
Hrm OK thats a new one to me.

@stgraber any idea where `target=none` is used? Is that still relevant?